binary-codec/definitions/definitions.json

```diff
@@ -30,34 +30,35 @@
     "Metadata": 10004
   },
   "LEDGER_ENTRY_TYPES": {
-    "Invalid": -1,
     "AccountRoot": 97,
-    "DirectoryNode": 100,
-    "RippleState": 114,
-    "Ticket": 84,
-    "SignerList": 83,
-    "Offer": 111,
-    "Bridge": 105,
-    "LedgerHashes": 104,
     "Amendments": 102,
-    "XChainOwnedClaimID": 113,
-    "XChainOwnedCreateAccountClaimID": 116,
-    "FeeSettings": 115,
-    "Escrow": 117,
-    "PayChannel": 120,
+    "AMM": 121,
+    "Any": -3,
+    "Bridge": 105,
     "Check": 67,
+    "Child": -2,
+    "Contract": 99,
+    "Credential": 129,
```

Comment:
This line only is new, the rest is sort alphabetically to better read the LEDGER_ENTRY_TYPES object.